debian/README.etc

```diff
@@ -0,0 +1,17 @@
+Configuring the Kolibri Debian package:
+
+Kolibri is normally configured in its own environment, for instance in
+its main configuration file `~/.kolibri/options.ini` or through command
+line options such as `kolibri start --port=1234`.
+
+If you want Kolibri to be configured in its external environment, you can
+set a number of environment variables.
+
+The prefered way to define them is to add your own file in
```

Comment:
what are the advantages of defining the variables here instead of using options.ini?
This is a system conf file. I understand it would be better in case several kolibri instances are running, but that's not possible with the current setup. `apt purge` will delete this file while options.ini is not, so I wonder why pushing for this alternative . I am sure there are good reasons I am not aware.

Reply:
The target group of these instructions are doing deployments that don't contain any `options.ini`, since that's a local configuration file for 1 environment. Adding something in `/etc/kolibri/conf.d/my_default_port` would be done because someone is doing a configuration for 10 deployments that all need to use that port but won't have `options.ini` distributed.

Reply:
> The target group of these instructions are doing deployments that don't contain any `options.ini`, since that's a local configuration file for 1 environment. `Adding something in /etc/kolibri/conf.d/my_default_port` would be done because someone is doing a configuration for 10 deployments that all need to use that port but won't have `options.ini` distributed.

How is creating `options.ini` different than creating a `conf.d` file?

Reply:
`options.ini` doesn't exist unless kolibri has been started in a known environment...  but if the kolibri user and KOLIBRI_HOME is unknown, then it's not a nice way to add an environment to Kolibri. We have all this debian-related configuration stuff because of that :) That is the defined and discussed environment that I wanted to add a README for. It already exists, so the README is intended as a formality rather than a design decision.

Am I missing something here?

Reply:
> if the kolibri user and KOLIBRI_HOME is unknown

sincere question: when would this be the case?

Reply:
sudo apt install kolibri.deb

Reply:
https://github.com/iiab/iiab/pull/2001/files#diff-73d8009e5abbed36f6d1eef2e412bb06R39

Reply:
I got around it here, but I don't think it's necessarily the preferred way of maintaining configuration.. I'd much rather have the conf.d style.. https://gitlab.com/fairdk/offline-digital-library/blob/master/src/conf.d/kolibri.sh#L46

Reply:
Okay I must be misunderstanding what you're saying because in both those scripts, it appears the kolibri user and home directory are known (`/home/kolibri/.kolibri` in [fair](https://gitlab.com/fairdk/offline-digital-library/blob/master/src/conf.d/kolibri.sh#L54) and `{{ kolibri_home }}` in [iiab](https://github.com/iiab/iiab/pull/2001/files#diff-73d8009e5abbed36f6d1eef2e412bb06R40)).

Doesn't really matter - just trying to get on the same page. Out of curiosity, is the text [in this PR](https://github.com/benjaoming/kolibri-installer-debian/pull/1/files) incorrect or misleading?

Reply:
Thanks for referencing your PR -- I don't understand why Github has my own fork marked as "not watching".